README.md

```diff
@@ -1,2 +1,30 @@
+# Dark theme for OSRS by iawhd
 
+To use this Resource Pack you must first install Runelite, as well as the external plugin '[Resource Packs](https://github.com/melkypie/resource-packs)' from the plugin hub.
 
+You must then download the latest zip file from the [releases page](https://github.com/iawhd/OSRS-Dark/releases), Extract it to a folder and then follow the instructions on the '[Resource Packs](https://github.com/melkypie/resource-packs)' plugin page.
```

Comment:
The https://github.com/iawhd/OSRS-Dark/releases should link towards https://github.com/melkypie/resource-packs/archive/pack-osrs-dark.zip so that the branch's version is used when downloading